app/controllers/AnnotationIOController.scala

```diff
@@ -281,10 +281,19 @@ Expects:
           .map(layer => elementClassToProto(layer.elementClass))
           .getOrElse(elementClassToProto(VolumeTracingDefaults.elementClass)),
         fallbackLayer = fallbackLayer.map(_.name),
-        largestSegmentId = fallbackLayer.map(_.largestSegmentId).getOrElse(VolumeTracingDefaults.largestSegmentId)
+        largestSegmentId =
+          combineLargestSegmentIds(volumeTracing.largestSegmentId, fallbackLayer.map(_.largestSegmentId))
       )
     }
 
+  private def combineLargestSegmentIds(fromNml: Option[Long], fromFallbackLayer: Option[Option[Long]]): Option[Long] =
```

Comment:
From this name I would have expected some max operation. Why exactly this precedence order, what is the use-case here?

Reply:
PS: The method `combineLargestSegmentIds` in `VolumeTracingService.scala` does what I assumed what this function would do.